### DISCUSSION TITLE
Ordering of Sidebar Navigation content

### DISCUSSION THREAD

**Discussion by dweng0** [quarto-dev/quarto-cli#7660] [Q&A] [answered]
### Description

Hi all, is there an explicit way (for example in the frontmatter) to force ordering of content on the sidebar navigation?

somthing like this in the frontmatter of all files in a given folder:
``` yaml
order: 1/2/3/4 
```

**Comment by mcanouil** [comment, marked as answer]
Specifying the order in each file for something global? This seems pretty laborious and prone to errors, *i.e.*, for each new content, you would need to be aware or all "order" you set everywhere before adding a new one and don't even think about reordering.

Why not just set the order directly in the sidebar when you fill it? You can use various ways and combine them to list and order the content.
https://quarto.org/docs/websites/website-navigation.html#side-navigation

**Reply by BrotherJiang** [reply]
Hi, I'm using Quarto to create my blog and I'm encountering an issue with sorting the contents in the sidebar. In my index.qmd file, I use the listing feature to sort the contents in the posts folder by date in descending order, which works perfectly. However, when I try to apply the same sorting logic in the _quarto.yml file for the sidebar, the sidebar fails to show up.

If I use contents: "posts/blogs/*.qmd" in the _quarto.yml file, the sidebar appears, but the file order is incorrect. As a workaround, I have been naming each file sequentially (e.g., 01blog1, 02blog2), but this is not ideal as it doesn't allow for sorting by date.

Could you please advise on how I can modify the _quarto.yml file to enable the sidebar to display the files sorted by date in descending order, similar to how it works on the homepage table using sort: date desc?

Thank you for your assistance.

**Reply by mcanouil** [reply]
@BrotherJiang could you open a new discussion and share a small reproducible example of your issue?

**Reply by BrotherJiang** [reply]
Hi, I have started a new discussion titled "Issue with Sidebar Sorting in Quarto Blog #9782". I hope you can help me with this issue. Thank you very much for your assistance.
